feat: Added Accordion in Tips Page

AianMarkAntonio opened the pull request:
Created Accordion and external stylesheet

AianMarkAntonio committed:
feat: Added Accordion in HTML form

Created Accordion and external stylesheet

Christiann12 reviewed (changes requested):
Add accordion to the actual tips php file. Rename css file to tips.css. Use Bootstrap Accordion.

AianMarkAntonio committed:
Merge branch 'main' into issue-34

AianMarkAntonio committed:
Merge branch 'main' into issue-34

AianMarkAntonio committed:
feat: Created FAQs page

Added link in the header, added link in the footer, created faqs.css and modified FAQ.php

Christiann12 reviewed (approved):
Works perfectly
